[JP] Grid topics localization fixes.

randriova opened the pull request.

HUSSAR-mtrela reviewed (changes requested):
please resolve conflicts and make sure you do not use:
{ComponentTitle}Component
instead just use:
{ComponentTitle}

randriova commented:
> please resolve conflicts and make sure you do not use: {ComponentTitle}Component instead just use: {ComponentTitle}

I've resolved the conflicts and replaced the {ComponentTitle}Component occurences. Additionaly, I applied the **{Platform} {ComponentTitle} for {ProductName}** in the title section to match the EN files and fixed the formatting on the JP version of `Change Theme`.
